src/components/BoardUsers/BoardUsers.tsx

```diff
@@ -0,0 +1,32 @@
+import BoardUser from './BoardUser';
+import RestUserCounter from './RestUsersCounter';
+import './BoardUsers.scss';
+import Parse from 'parse';
+import { useSelector } from 'react-redux';
+import { ApplicationState } from 'types/store';
+
+export interface BoardUsersProps {
+  numOfUsersToShow: number;
+}
+
+const BoardUsers = ({ numOfUsersToShow }: BoardUsersProps) => {
+  const users = useSelector((state: ApplicationState) => state.users.all);
+  const currentUser = Parse.User.current();
+
+  const me = users.find(user => user.id === currentUser!.id);
+  const connectedUsers = users.filter(user => user.online);
+  const them = connectedUsers.filter(user => user.id !== currentUser!.id);
+  const usersToShow = them.splice(0, (them.length > numOfUsersToShow ? numOfUsersToShow - 1 : numOfUsersToShow));
```

Comment:
Also here you should use the `connectedUsers` instead of `them` and combine the conditional of both queries `(user.id !== currentUser.id && user.online)`